lib/src/components/Flex/flex.types.ts

```diff
@@ -1,12 +1,16 @@
-export interface FlexProps extends React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>
-{
-    alignContent?: React.CSSProperties['alignContent'],
-    alignItems?: React.CSSProperties['alignItems'],
-    flexDirection?: React.CSSProperties['flexDirection'],
-    flexWrap?: React.CSSProperties['flexWrap'],
-    justifyContent?: React.CSSProperties['justifyContent'],
-    flexShrink?: React.CSSProperties['flexShrink'],
-    flexGrow?: React.CSSProperties['flexGrow'],
-    flexBasis?: React.CSSProperties['flexBasis'],
-    gap?:React.CSSProperties['gap'],
-}
\ No newline at end of file
+export interface FlexProps
+  extends React.DetailedHTMLProps<
+    React.HTMLAttributes<HTMLDivElement>,
+    HTMLDivElement
+  > {
+  alignContent?: React.CSSProperties["alignContent"];
+  alignItems?: React.CSSProperties["alignItems"];
+  flexDirection?: React.CSSProperties["flexDirection"];
+  flexWrap?: React.CSSProperties["flexWrap"];
+  justifyContent?: React.CSSProperties["justifyContent"];
+  flexShrink?: React.CSSProperties["flexShrink"];
+  flexGrow?: React.CSSProperties["flexGrow"];
+  flexBasis?: React.CSSProperties["flexBasis"];
+  gap?: React.CSSProperties["gap"];
+  display?: "flex" | "inline-flex";
```

Comment:
okay okay !! I didn't know there was something inline-flex in the display property.
Got it now !!

Reply:
Cool then Merging, awesome seeing improvements on components we are using 😄